lighthouse-core/aggregator/aggregate.js

```diff
@@ -77,6 +77,12 @@ class Aggregate {
       throw new Error(msg);
     }
 
+    // Audit resulted in an error, so doesn't contribute to score.
```

Comment:
> Basically, users should never see `NaN`. That looks like a bug

one question I'm immediately running into is what *should* it display in the report for these error results. In theory I could continue doing -1 for the numeric ones, but that's not great, and what about the boolean ones?

Could do red X from failing audits, greyed out icon from the coming soon audits, some kind of question mark icon, ..? Anyone have a good idea? Doesn't have to be our symbol for life :)